projects/kit/components/tabs/tabs/tabs.style.less

```diff
@@ -21,4 +21,11 @@
         background: var(--tui-link);
         box-shadow: none;
     }
+
+    &::ng-deep > :where(:not(:last-child)) {
+        & [tuiTab],
+        &[tuiTab] {
```

Comment:
awful solution :(

Guys, do you have any ideas how to improve it ?

Reply:
maybe just add compensation in tabs-with-more? or problem is appears not only in tabs-with-more ?

Reply:
Current solution works. But I agree that it is awful(

Reply:
Can we write e2e tests for `example, [tuiTab] in wrapped by hosted-dropdown`? 
and add `TODO: find the better way.` for `&::ng-deep`

Reply:
@splincode 
> Can we write e2e tests for example, [tuiTab] is wrapped by hosted-dropdown

we have already this test (`demo.spec.ts`)

Reply:
```
But we don't have the test for this bug)
I will add test for this case 👌
```

Thank you))